block/transformer.go

```diff
@@ -0,0 +1,6 @@
+package block
+
+// Transformer transforms parsed observer data. For example decrypt, or transformer protobuf state to json
+type Transformer interface {
+	Transform(*Block) (*Block, *Block, error)
```

Comment:
why 2 *Block returned ?

Reply:
first is transformed, second - original. Will add comment